assets/style.css

```diff
@@ -0,0 +1,603 @@
+  text-decoration: none;
+}
+
+.footer-bottom a {
+  display: inline-block;
+  text-decoration: none;
+  padding: 0 5px;
+  font-size: 11px;
+  color: #666;
+  font-weight: 400;
+}
+
+.footer-heading a {
+  font-size: 13px;
+  text-decoration: none;
+  color: #567b95;
+}
+
+.editor-pick-aside a {
+  color: #333;
+  text-decoration: none;
+  font-size: 14px;
+  line-height: 17px;
+  font-weight: 100;
+}
+
+.content span a {
+  font-weight: 400;
+  color: #326891;
+  text-decoration: underline;
+}
+
+.main-content .author-name a {
+  text-decoration: underline;
+  text-decoration-color: rgba(180, 177, 177, 0.589);
+}
+
+.space-astronomy a:hover {
+  text-decoration: underline;
+  cursor: pointer;
+}
+
+.most-popular a:hover {
+  color: #666;
+}
+
+.footer-bottom a:hover {
+  text-decoration: underline;
+}
+
+.editor-pick-aside a:hover {
+  text-decoration: underline;
+}
+
+.main-content p a:hover {
+  cursor: pointer;
+  text-decoration: none;
+}
+
+.date-share-icons p {
+  font-size: 13px;
+}
+
+.date-share-icons ul {
+  flex: 70%;
+  justify-content: flex-end;
+}
+
+.social-icons {
+  display: inline-block;
+  margin: 0 9px;
+  font-size: 14px;
+  background-color: rgba(209, 208, 208, 0.623);
+  padding: 6px;
+  border-radius: 50%;
+  border: 1px solid rgba(209, 208, 208, 0.623);
+}
+
+.fa-facebook {
+  padding: 6px 9px;
+}
+
+.fa-bookmark,
+.fa-comment-alt {
+  font-size: 25px;
+  padding: 0 7px;
+  color: rgba(0, 0, 0, 0.603);
+  display: inline-block;
+  font-weight: 100;
+}
+
+.book-comment {
+  font-size: 50px;
+  font-weight: 400;
+  line-height: 20px;
+  margin: 0 10px;
+}
+
+.date-share-icons .social-icons:hover {
+  background-color: #fff;
+  cursor: pointer;
+}
+
+.theory-of-inflation {
+  width: 70%;
+  margin: 0 auto;
+}
+
+.theory-of-inflation h4,
+.theory-of-inflation p {
+  font-size: 16px;
+}
+
+.theory-of-inflation h4 {
+  margin-bottom: -8px;
+}
+
+.theory-of-inflation p {
+  font-size: 14px;
+  line-height: 16px;
+  color: rgb(107, 103, 103);
+}
+
+.menu li {
+  font-size: 14px;
+  padding-bottom: 10px;
+}
+
+.subscribe-section li {
+  padding-bottom: 15px;
+  padding-left: -20px;
+}
+
+.first-row ul li,
+.second-row ul li {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  justify-content: space-between;
+}
+
+.first-row ul,
+.second-row ul {
+  margin: 0;
+  padding: 0;
+}
+
+.telescope-img figcaption {
+  color: #666;
+  font-size: 17px;
+  line-height: 20px;
+  width: 97%;
+  padding-top: 8px;
+}
+
+.space-astronomy figure,
+.editor-pick figure {
+  margin-block-start: 0;
+  margin-block-end: 0;
+  margin-inline-start: 0;
+  margin-inline-end: 0;
+}
+
+.first-row ul li figure,
+.second-row ul li figure {
+  margin-left: 0;
+  padding-right: 10px;
+}
+
+.footer-heading {
+  justify-content: space-between;
+}
+
+.space-astronomy figure figcaption,
+.editor-pick figure figcaption {
+  width: 290px;
+}
+
+.first-row ul li figure figcaption,
+.second-row ul li figure figcaption {
+  font-size: 13px;
```

Comment:
## Using Inefficient CSS Selectors[1]

When you are using selectors, the longer it is, the higher specificity it will comprise. This can break the CSS cascade and mess up the styles of your site.

This can be avoided by not using IDs within the selectors as they are directly specific.

 - [ ] Please add a class to the `<figcaption>`  element and avoid use like so

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_